rfcs/2-jhipster-rfc-jhipster-companion.md

```diff
@@ -0,0 +1,142 @@
+- Feature Name: `jhipster_companion`
+- Start Date: 2019-12-12
+- Issue: [jhipster/generator-jhipster#10913](https://github.com/jhipster/generator-jhipster/issues/10913)
+
+
+## Table of content
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Guide-level explanation](#guide-level-explanation)
+  - [Architecture overview](#architecture-overview)
+  - [Description of features](#description-of-features)
+  - [Usage in development](#usage-in-development)
+  - [Usage in production](#usage-in-production)
+- [Reference-level explanation](#reference-level-explanation)
+- [Drawbacks](#drawbacks)
+- [Rationale and alternatives](#rationale-and-alternatives)
+- [Prior art](#prior-art)
+- [Unresolved questions](#unresolved-questions)
+- [Future possibilities](#future-possibilities)
+
+# Summary
+
+[summary]: #summary
+
+As of JHipster 6.5, the JHipster administrator's management and monitoring UI is spread out to various places (JHipster Registry, angular/react/vue.js pages) which causes code duplication and restraints their evolution. This RFC proposes to move this functionality to an external application that can be used in different development and production scenarios. In the following, this application will be referred to as the **JHipster companion**.
```

Comment:
restraints => restrains
maybe it's too "heavy" a meaning for what you meant, what about "slows down"? Strong enough?